src/ucm/cuda/cudamem.c

```diff
@@ -206,13 +210,38 @@ UCM_CUDA_FREE_FUNC(cuMemFreeAsync, UCS_MEMORY_TYPE_CUDA, CUresult, arg0, 0,
                    "ptr=0x%llx, stream=%p", CUdeviceptr, CUstream)
 #endif
 
+CUresult ucm_cuModuleGetGlobal_v2(CUdeviceptr *dptr, size_t *bytes,
+                                  CUmodule hmod, const char *name)
+{
+    CUresult ret;
+    size_t size;
+    size_t *size_ptr;
+
+    if (dptr == NULL) {
+        return ucm_orig_cuModuleGetGlobal_v2(dptr, bytes, hmod, name);
+    }
+
+    size_ptr = (bytes == NULL) ? &size : bytes;
+    ucm_event_enter();
+    ret = ucm_orig_cuModuleGetGlobal_v2(dptr, size_ptr, hmod, name);
+    if (ret == CUDA_SUCCESS) {
+        ucm_trace("%s(bytes=%p, hmod=%p, name=%s) returned %p", __func__, bytes,
+                  hmod, name, (void*)(*dptr));
+        ucm_cuda_dispatch_mem_alloc(*dptr, *size_ptr);
+    }
+
```

Comment:
minot: remove space line

Reply:
But we have the recommendation in `The UCX code style` to use blank line after if/else blocks.

Reply:
it seems inconsistent with ucm_event_enter

Reply:
Removed.